cfme/tests/automate/test_method.py

```diff
@@ -842,3 +843,37 @@ def test_copy_with_embedded_method(request, appliance, klass):
     )
     view = navigate_to(copied_method, "Details")
     assert view.embedded_method_table.read()[0]["Path"] == embedded_method_path
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(blockers=[BZ(1744514)], automates=[1744514])
+@pytest.mark.customer_scenario
+def test_delete_tag_from_category(custom_instance):
```

Comment:
**Suggestion:**
Whole test is around CRUD via Automate method, test case name should be something like: `test_automate_method_delete_tag_from_category` or `test_automate_delete_tag_from_category` to have consistency around it.

Reply:
I don't think we need to change test name. It is already under in automate  related files. hence keeping as it is.

Reply:
Yes, @ganeshhubale I agree with you about the test is present in the same file, but looking all of your tests in the same file have suggested name, implementation is up to you :smile:

Reply:
As long as there isn't duplication of the test case names across test modules, I think the naming here is fine, as some context is given by the module.